### PR TITLE
backend/excel/export: Support Excel Export of custom grammar fields

### PR DESCRIPTION
**Pull request by GGBeer**

### CODE REVIEW

File: tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest [hunk 1]
```diff
@@ -0,0 +1,5 @@
+RUN: %strictdoc export --formats=excel --fields=uid,owner --output-dir=Output "%S/input.sdoc"
```
Comment (stanislaw) [inline tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest:1]: I have checked the expected Excel file manually, and it looks like you are not actually testing most of the things that you have in the `input.sdoc`. Could you remove the `--fields` argument, regenerate the Excel file, make sure that it looks like as expected, and finally force-push your change?

**Review by stanislaw — CHANGES_REQUESTED** [review]
The changeset looks like going to the right direction overall, but it would be great if you strengthened the integration tests.

File: tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest (hunk 1 above)
Comment (GGBeer) [reply tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest]: Added some new Export Excel ITests. Testing showed that the specials "Parent" and "Comments" needed some work.

Comment (GGBeer) [reply tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest]: The new PR failed the ITest "07_custom_grammar_fields_to_export_fields_all_special" in the "Periodic Integration Test" 
Seems that pybtex somewhre needs the Six package? Not sure where to include/fix this.

Comment (stanislaw) [reply tests/integration/backend/excel/export/04_custom_grammar_fields_to_export/test.itest]: That part of the integration test has to do with a rather tricky setup of PyInstaller. I am hoping to take a look into this still today.

**Review by stanislaw — APPROVED** [review]

**Comment by stanislaw** [comment]
Thank you again for the contribution! I am merging this as a good step forward. In the near future, I think, it would be great to discuss the improvements of the layout of the produced Excel table. I feel like we would rather make it be formatted with font sizes like a document and somehow move to the "1 field per 1 row" approach. I don't have a clear visual solution to this yet, but some improvement is definitely needed.

Furthermore, similar to ReqIF, it would be great to add the "roundtrip" integration tests - an .sdoc is exported to an Excel file, then imported back, and you get a 100% original .sdoc again. It's something that's been on my list for a few months but the time was not there 😞

**Pull request merged**